### DISCUSSION TITLE
Can The Current V1.3.3 Release Be Updated?

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#944] [General Discussion]
@Jyers  Others are having issues with the Pause feature not working with the precompiled files. Can these be updated with the Pause Timeout fix?

**Comment by Jyers** [comment]
It's generally not good practice to a change a release. For now, anyone worried about timeout issues can use the previous release and a new release containing the fix will be out in at most a few days.

**Reply by hkasap83** [reply]
is it possible to move the nozzle a little auto side or up when the power loss? i think everbody has an experience that when the power comes, nozzle becomes hot to resume printing but it also melts that position of the part.

**Reply by Jyers** [reply]
Powerloss recovery is a marlin feature, if you would like to suggest an improvement, I recommend posting a feature request on the main marlin repo
